recipes/monetdb/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "MonetDB" %}
+{% set version = "11.25.15" %}
+{% set sha256 = "8166268f676f75f0df7fdf3f8b2a49ffff422420d275bb994dddcab95012f89e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://www.monetdb.org/downloads/sources/Dec2016-SP3/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - toolchain
+    - pcre
+    - libxml2
```

Comment:
Please pin this to `libxml2 2.9.*` in build and run requirements. [full pinnings are listed here](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L28-L71)